game-core/src/main/java/games/strategy/triplea/ui/AbstractMovePanel.java

```diff
@@ -238,10 +238,7 @@ protected final void display(final GamePlayer gamePlayer, final String actionLab
         () -> {
           removeAll();
           add(movedUnitsPanel(gamePlayer, actionLabel));
-          add(
-              getUnitScrollerPanel(
-                  frame.getLocalPlayers(),
-                  () -> FlagDrawMode.toggleNextDrawMode(frame.getMapPanel())));
```

Comment:
Is this method still used?

Reply:
AFAIK we did not remove any references to a `display(` method in this PR, please note if I'm wrong about that. Given that, the method should still be used if it was used before at all.